client/src/components/brushableHistogram/index.js

```diff
@@ -476,6 +476,8 @@ class HistogramBrush extends React.PureComponent {
     const unclippedRangeMaxColor =
       world.clipQuantiles.max === 1 ? "#bbb" : globals.blue;
 
+    const isSingleValue = unclippedRangeMin === unclippedRangeMax;
```

Comment:
what behavior are you targeting when the user has clipped the range, so that it is all single-valued or NaN (clipped)?   In this case, the NaN doesn't show up in the histogram, and the histogram will end up being single-valued.

So shouldn't the conditional be based upon the min/max of the clipped range, rather than the unclipped range?